packages/syft/tests/syft/lib/python/slice/slice_test.py

```diff
@@ -124,6 +141,17 @@ def __eq__(self, other):
         self.assertEqual(s1, s1)
         self.assertRaises(Exc, lambda: s1.value == s2.value)
 
+    def test_id(self):
+        new_id = UID()
+        s1 = Slice(1, 2, 3, id=new_id)
+        assert new_id == s1.id
+
+    def test_upcast(self):
```

Comment:
Q: Should we also have a test when we test a `slice` against a `Slice`? (to see that the `__eq__` from slice works correctly?)

Reply:
Yeah, good point!